mycore-base/src/main/java/org/mycore/datamodel/common/MCRXMLMetadataManager.java

```diff
@@ -326,36 +326,36 @@ public MCRMetadataStore getStore(String project, String type, boolean readOnly)
     @SuppressWarnings("unchecked")
     private void setupStore(String project, String objectType, String configPrefix, boolean readOnly)
         throws ReflectiveOperationException {
-        MCRConfiguration config = MCRConfiguration.instance();
         String baseID = getStoryKey(project, objectType);
-        Class clazz = config.getClass(configPrefix + "Class", null);
-        if (clazz == null) {
-            config.set(configPrefix + "Class", defaultClass.getName());
-            clazz = defaultClass;
-        }
+        Class<? extends MCRStore> clazz = MCRConfiguration2.<MCRStore> getClass(configPrefix + "Class")
+            .orElseGet(() -> {
+                MCRConfiguration2.set(configPrefix + "Class", defaultClass.getName());
+                return defaultClass;
+            });
         if (MCRVersioningMetadataStore.class.isAssignableFrom(clazz)) {
             String property = configPrefix + "SVNRepositoryURL";
-            String svnURL = config.getString(property, null);
+            String svnURL = MCRConfiguration2.getString(property).orElse(null);
             if (svnURL == null) {
                 String relativeURI = new MessageFormat("{0}/{1}/", Locale.ROOT)
                     .format(new Object[] { project, objectType });
                 URI repURI = svnBase.resolve(relativeURI);
                 LOGGER.info("Resolved {} to {} for {}", relativeURI, repURI.toASCIIString(), property);
-                config.set(property, repURI.toASCIIString());
+                MCRConfiguration2.set(property, repURI.toASCIIString());
                 checkAndCreateDirectory(svnPath.resolve(project), project, objectType, configPrefix, readOnly);
             }
         }
 
         Path typePath = basePath.resolve(project).resolve(objectType);
         checkAndCreateDirectory(typePath, project, objectType, configPrefix, readOnly);
 
-        String slotLayout = config.getString(configPrefix + "SlotLayout", null);
+        String slotLayout = MCRConfiguration2.getString(configPrefix + "SlotLayout").orElse(null);
         if (slotLayout == null) {
-            config.set(configPrefix + "SlotLayout", defaultLayout);
+            MCRConfiguration2.set(configPrefix + "SlotLayout", defaultLayout);
         }
-        config.set(configPrefix + "BaseDir", typePath.toAbsolutePath().toString());
-        config.set(configPrefix + "ForceXML", true);
-        config.set(configPrefix + "ForceDocType", objectType.equals("derivate") ? "mycorederivate" : "mycoreobject");
+        MCRConfiguration2.set(configPrefix + "BaseDir", typePath.toAbsolutePath().toString());
+        MCRConfiguration2.set(configPrefix + "ForceXML", String.valueOf(true));
+        String value = objectType.equals("derivate") ? "mycorederivate" : "mycoreobject";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Position literals first in String comparisons](https://app.codacy.com/manual/MyCoRe/mycore/pullRequest?prid=5299284)